### ISSUE TITLE
jsrender missing whitespace in rendered output when template expression immediately follows opening table element

### ISSUE THREAD

**Issue by johan-ohrn** [BorisMoore/jsviews#415]
Look at this [fiddle](https://jsfiddle.net/u5m2kg19/3/) with the console open and inspect the generated html code.
This output from this template is missing a whitespace between tag and the attribute:
```
<table {{if 1}} data-whatever{{/if}}>
</table>
```
it's rendered as:
```
<tabledata-whatever>
</tabledata-whatever>
```
Expected rendered output is:
```
<table data-whatever>
</table>
```

Even manually adding whitespsace inside the if-expression doesn't help.
I'm only having this issue with the table tag. Divs and other elements seem to work as expected.

**Comment by BorisMoore** [comment]
Another excellent catch. Thank you!

Here is a proposed fix. It is a one-liner in jsviews.js or jquery.views.js
[download.zip](https://github.com/BorisMoore/jsviews/files/2862498/download.zip)

```
if (elCnt && !inTag) {
	// elContent maps tagNames which have only element content...
```

Let me know if it works correctly for you.

**Comment by johan-ohrn** [comment]
I can confirm that it works.
Thanks a lot!

**Comment by BorisMoore** [comment]
The fix is included in the new v1.0.3 release.
